### ISSUE TITLE
eLAN-RF-103

### ISSUE THREAD

**Issue by Geometric4353** [JH-Soft-Technology/InelsForHass#17]
Hello, does the plugin also work with eLAN-RF-103?
If yes, where can I enter the user and password?
At the moment I only get Unexpected error.
The port 9999 is also used by the Inels software.
thanks for an answer
![1](https://github.com/JH-Soft-Technology/InelsForHass/assets/165678329/87a8d4b0-5963-4dd6-bfa8-240580718691)

## Configuration

```yaml

Add your logs here.

```

## Debug log

```text

Add your logs here.

```

**Comment by ds141414** [comment]
Hi, you may want to check this out

https://github.com/zdar/elan2mqtt/

It works well with Home Assistant, I have been using it for a while

**Comment by jhoralek** [comment]
Hi @Geometric4353,

no it is working only on BUS. 

Wireless devices working over the eLan which is mentioned with @ds141414.

Official attempt to integrate iNELS device by Elko itself died last year. I was part of the project at the beginning, but after a certain time I was removed from it in an unprofessional way.

@ds141414 I'll give a shot to this project. . I've got old eLan hub with some bulbs. 

Closing this ticket.
